traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+Modeling traffic participants
+==============================
+
+Traffic participants
+--------------------
+
+A traffic participant is an element of the simulated world, which can change its position and orientation during simulation time.
+A traffic participant represents one of the following:
+
+- A living being.
+- A means of transportation for living beings
+- A means of transportation for goods
+- Any other movable object that may travel the road network
+
+Pedestrians and animals are examples of traffic participants that are living beings.
+Vehicles are examples of traffic participants that are means of transportation.
+Therefore, the ego vehicle is also a traffic participant.
+
+The following figure shows the interface of a traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-principle.png
+   :align: center
+   :alt: Traffic participant interfaces
+
+With every simulation step, an OSI traffic participant model receives ground truth data from the environment around itself, the sensor view.
+A traffic participant can output its own perceivable state, the traffic update.
+Traffic commands influence the behavior of the traffic participant model.
+Traffic commands allow event-based communication towards the traffic participant, for example, at certain simulation steps.
```

Comment:
"i.e." instead of "for example"

Reply:
- [ ] Exchange "for example" for "that is".

Background: In the past, we avoided "i.e." and "e.g.". These two abbreviations are a common source of misinterpretation (that's what's happend to me up there) and inconsistent spelling ("i. e." vs. "i.e.").